src/preferences/general.hpp

```diff
@@ -274,4 +274,7 @@ namespace preferences {
 	sort_order::type addon_manager_saved_order_direction();
 	void set_addon_manager_saved_order_direction(sort_order::type value);
 
+	bool achieved(const std::string& content_for, const std::string& id);
+	void set_achieved(const std::string& content_for, const std::string& id);
```

Comment:
The names are a little ambiguous - `achieved` is a verb in the past tense, so it could be the thing that means "set achieved".

Just because there's no documentation in the file so far doesn't mean that's a good thing. We should set an image for other developers to follow :grin:

Reply:
`has_achieved` would be better, yeah. Maybe `is_achieved`.

Reply:
I followed the naming convention used in the rest of the file though.

Reply:
The rest of the file is using nouns as the getters' names, so maybe `achievement` and `set_achievement`.

Reply:
Updated.